CMake/FindMKL.cmake

```diff
@@ -27,20 +43,13 @@ if ( NOT CMAKE_CXX_COMPILER_ID MATCHES "Intel" )
       "not found in MKL_ROOT/(${SUFFIXES})")
   endif (MKL_LINK_DIRECTORIES-NOTFOUND)
   message("MKL_LINK_DIRECTORIES: ${MKL_LINK_DIRECTORIES}")
-
-  # Finding and setting the MKL_INCLUDE_DIRECTORIES
-  set(SUFFIXES include)
-  find_path(MKL_INCLUDE_DIRECTORIES name "mkl.h" HINTS ${MKL_ROOT}
-    PATH_SUFFIXES ${SUFFIXES})
-  if (MKL_INCLUDE_DIRECTORIES-NOTFOUND)
-    message(FATAL_ERROR "MKL_INCLUDE_DIRECTORIES not set. \"mkl.h\" not found in MKL_ROOT/(${SUFFIXES})")
-  endif (MKL_INCLUDE_DIRECTORIES-NOTFOUND)
-  message("MKL_INCLUDE_DIRECTORIES: ${MKL_INCLUDE_DIRECTORIES}")
-
   set(MKL_LINKER_FLAGS "-L${MKL_LINK_DIRECTORIES} -Wl,-rpath,${MKL_LINK_DIRECTORIES}")
   set(MKL_LIBRARIES "-lmkl_intel_lp64 -lmkl_sequential -lmkl_core -lpthread -lm -ldl")
 else ( NOT CMAKE_CXX_COMPILER_ID MATCHES "Intel" )
-  # this takes away build control for intel but is convenient
+  # this takes away link control for intel but is convenient
+  # perhaps we should consider dropping it since it will more or less
```

Comment:
The non Intel compiler routine hard coded sequential MKL but threaded MKL is better. -mkl picks the threaded one.

Reply:
Shouldn't we change that then?

Reply:
The conclusion from the last discussion was leaving it sequential for non intel compilers.